Doc/c-api/stable.rst

```diff
@@ -18,8 +18,35 @@ way; see :ref:`stable-abi-platform` below).
 So, code compiled for Python 3.10.0 will work on 3.10.8 and vice versa,
 but will need to be compiled separately for 3.9.x and 3.10.x.
 
+There are two tiers of API with different stability exepectations,
+enabled by specific macros:
+
+- :c:macro:`Py_USING_SEMI_STABLE_API` exposes API that may change
+  without deprecation warnings.
+- :c:macro:`Py_LIMITED_API` exposes API that is compatible across
+  several minor releases.
```

Comment:
exepectations -> expectations

One could also say: "there are ~two~ _public_ tiers of the API" (emphasis just to show addition); underscored resp. `Py_BUILD_CORE` are arguably a tier as well...?

I've struck out the "two" because I think it would be clearer to expose the full hierarchy, à la:
```suggestion
There are three public tiers of the API with different stability
expectations, two of which need to be enabled by specific macros:

- :c:macro:`Py_LIMITED_API` exposes API that is compatible across
  several minor releases.
- (no macro): default C-API covered by :pep:`387`.
- :c:macro:`Py_USING_SEMI_STABLE_API` exposes API that may change
  without deprecation warnings between minor releases.
```

I'd also suggest to lead with the most stable one.

Reply:
By “There are two tiers of API *with different stability expectations*” I wanted to present them as being additional, different from the default that's covered in the preceding paragraphs.
The order matches the detailed sections below, which would be somewhat tricky to reorder -- the one-paragraph semi-stable tier would feel lost if it was at the end.